noir-projects/noir-contracts-comp-failures/contracts/event_selector_collision/src/main.nr

```diff
@@ -0,0 +1,12 @@
+use aztec::macros::aztec;
+
+#[aztec]
+pub contract EventSelectorCollision {
+    use aztec::macros::events::event;
+
+    #[event]
+    struct EventCollision {}
+
+    #[event]
+    struct EventCollision8370082250 {}
```

Comment:
Finding this collision became a bit of a time sink 🤣 

Luckily @xorsal saved me by sharing his code.